build.props

```diff
@@ -4,11 +4,11 @@
     <EnlistmentRoot>$(MSBuildThisFileDirectory)</EnlistmentRoot>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
     <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <BaseIntermediateOutputPath Condition=" '$(BaseIntermediateOutputPath)'=='' ">$(EnlistmentRoot)\obj</BaseIntermediateOutputPath>
+    <BaseIntermediateOutputPath Condition=" '$(BaseIntermediateOutputPath)'=='' ">$(EnlistmentRoot)obj</BaseIntermediateOutputPath>

```

Comment:
for a maintainability aspect having to reason about the context to determine the property value adds additional confusion/risk.  I recommend doing a trim on this property as well so all references to this property will be the same so you don't need to reason if the cbt props were imported or not yet.  This issue I worry about is this build.props others will likely copy as a starting place and their build.props could grow in complexity making the reasoning more than trivial.